dotnet/src/Connectors/Connectors.Memory.Redis/RedisVectorIndexAlgorithms.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+namespace Microsoft.SemanticKernel.Connectors.Memory.Redis;
+
+/// <summary>
+/// Vector similarity index algorithm. Supported algorithms are {FLAT, HNSW}. The default value is "HNSW".
+/// <see href="https://redis.io/docs/interact/search-and-query/search/vectors/#create-a-vector-field"/>
+/// </summary>
+public enum VectorIndexAlgorithms
```

Comment:
DO use a singular type name for an enumeration unless its values are bit fields.
https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-classes-structs-and-interfaces#naming-enumerations